lib/git.zsh

```diff
@@ -77,8 +77,8 @@ function git_current_branch() {
 # Gets the number of commits ahead from remote
 function git_commits_ahead() {
   if command git rev-parse --git-dir &>/dev/null; then
-    local commits="$(git rev-list --count @{upstream}..HEAD)"
-    if [[ "$commits" != 0 ]]; then
+    local commits="$(git rev-list --count @{upstream}..HEAD 2>/dev/null)"
+    if [[ -n $commits ]] && [[ "$commits" != 0 ]]; then
```

Comment:
Any special reason you don't write:
``` if [[ -n "$commits" -a "$commits" != 0 ]]``` ?

IMHO putting the '"' around any tested environment variable is **always** a very good idea and using '-a' instead of terminating one test and running after that another one should speed up the thing a little...